pennylane/operation.py

```diff
@@ -226,7 +226,7 @@ class Operator(abc.ABC):
     * :attr:`~.Operator.par_domain`
 
     Args:
-        params (tuple[float, int, array, Variable]): operator parameters
+        data (tuple[float, int, array, Variable]): operator parameters
```

Comment:
@yishairasowsky: this is an argument to the Operation class, not the `Operation.params` attribute, so should remain as it is 🙂